shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -749,17 +749,34 @@ - (void)onMemoryWarning:(NSNotification*)notification {
 - (void)onLocaleUpdated:(NSNotification*)notification {
   NSArray<NSString*>* preferredLocales = [NSLocale preferredLanguages];
   NSMutableArray<NSString*>* data = [NSMutableArray new];
```

Comment:
So long as you're here, this should be `NSMutableArray<NSString*>* data = [[NSMutableArray new] autorelease];`.  I have another PR that fixes it, but might as well get it now.